src/main/InstanceService.ts

```diff
@@ -178,11 +178,11 @@ class InstanceService {
     }
   }
 
-  async sendTyping(instanceId: string, conversationId: string, payload: 'started' | 'stopped'): Promise<string> {
+  async sendTyping(instanceId: string, conversationId: string, status: 'started' | 'stopped'): Promise<string> {
     const instance = this.getInstance(instanceId);
 
     if (instance.account.service) {
-      if (payload === 'started') {
+      if (status === 'started') {
```

Comment:
Can you reuse `CONVERSATION_TYPING.STARTED` (from `api-client/src/main/conversation/Typing`) here?